static/js/src/ua-payment-modal.js

```diff
@@ -942,7 +942,8 @@ function setCustomerInfo() {
   customerInfo.email = formData.get("email");
   customerInfo.country = formData.get("Country");
   customerInfo.name = formData.get("name");
-  customerInfo.accountName = formData.get("account_name");
+  customerInfo.accountName =
+    formData.get("account_name") || formData.get("name");
```

Comment:
Should we trim the account name before doing the check?

Reply:
> Looks good. I'd also change the description for the account name field: "This can be your organisation, or your name again". We should make it clearer that the field is not required, nor is required to repeat your name again.

Do you know who I should ask to get a new copy?

Reply:
You could ask @matthewpaulthomas